app/assets/stylesheets/views/_html-publication.scss

```diff
@@ -0,0 +1,84 @@
+.html-publication {
+
+  .govuk-title .context {
```

Comment:
I don't think we should be relying on the internal markup of a component, that can change independently of this application.

The risk of this approach is that changes to the component, in markup/selectors, or explicitly setting colours on elements that don't have them could break this page without static being aware of the dependency. And it's generally a pattern we try and avoid. 

This feels more like a mode of the component and the responsibility for a white links/different margin version should live within the component, not the app. 

Alternatively, we treat it as a non standard page title, and don't use the component at all, setting _all_ the styles locally.